core-rust/state-manager/src/protocol/protocol_updates/definitions/bottlenose_definition.rs

```diff
@@ -0,0 +1,44 @@
+use crate::engine_prelude::*;
+use crate::protocol::protocol_updates::definitions::ScryptoEntriesBatchGenerator;
+use crate::protocol::*;
+
+const BOTTLENOSE_ENTRIES: [(&str, ProtocolUpdateEntry); 5] = [
+    (
+        "bottlenose-owner-role-getter",
+        ProtocolUpdateEntry::OwnerRoleGetter,
+    ),
+    (
+        "bottlenose-system-patches",
+        ProtocolUpdateEntry::SystemPatches,
+    ),
+    (
+        "bottlenose-locker-package",
+        ProtocolUpdateEntry::LockerPackage,
+    ),
+    (
+        "bottlenose-account-try-deposit-or-refund",
+        ProtocolUpdateEntry::AccountTryDepositOrRefundBehaviorChanges,
+    ),
+    (
+        "bottlenose-protocol-params-to-state",
+        ProtocolUpdateEntry::ProtocolParamsToState,
```

Comment:
:D Can I put you in charge of regularly updating this branch; and also of double-checking that all the bottlenose updates are pulled through before we do the actual release?

Reply:
yup, I will take care of this 👍